kratos/geometries/hexahedra_3d_8.h

```diff
@@ -1450,8 +1450,8 @@ template<class TPointType> class Hexahedra3D8 : public Geometry<TPointType>
                 Quadrature < HexahedronGaussLegendreIntegrationPoints4, 3, IntegrationPoint<3> >::GenerateIntegrationPoints(),
                 Quadrature < HexahedronGaussLegendreIntegrationPoints5, 3, IntegrationPoint<3> >::GenerateIntegrationPoints(),
                 Quadrature < HexahedronGaussLobattoIntegrationPoints1, 3, IntegrationPoint<3> >::GenerateIntegrationPoints(),
-                Quadrature < HexahedronGaussLobattoIntegrationPoints2, 3, IntegrationPoint<3> >::GenerateIntegrationPoints(),
-                Quadrature < HexahedronGaussLobattoIntegrationPoints2, 3, IntegrationPoint<3> >::GenerateIntegrationPoints()
+                Quadrature < HexahedronGaussLobattoIntegrationPoints1, 3, IntegrationPoint<3> >::GenerateIntegrationPoints(),
```

Comment:
```suggestion
```
Why repeated?

Reply:
It is not "repeated" actually. Previously, in the `EXTENDED_GI_GAUSS `thy were using the Lobatto so I just maintained the previous code.